src/js/custom.js

```diff
@@ -1,5 +1,5 @@
 // Start scroll infinite, popup and script echo.
-$(document).ready(function(){
+$(function(){
```

Comment:
I think this update has already been done: https://github.com/femontanha/free-time.github.io/blob/master/src/js/custom.js